egs/wsj/s5/steps/libs/nnet3/train/frame_level_objf/common.py

```diff
@@ -636,10 +722,10 @@ def adjust_am_priors(dir, input_model, avg_posterior_vector, output_model,
                     avg_posterior_vector=avg_posterior_vector,
                     output_model=output_model))
 
-
 def compute_average_posterior(dir, iter, egs_dir, num_archives,
                               prior_subset_size, left_context, right_context,
-                              run_opts, get_raw_nnet_from_am=True):
+                              run_opts, get_raw_nnet_from_am=True,
+                              use_multitask_egs=False):
```

Comment:
Actually we use train_raw_dnn.py and we don't use this function. We separately rename output-name to "output" and compute average posteriors and adjust priors.